src/xpcc/architecture/platform/driver/gpio/stm32f1/gpio_enable.cpp.in

```diff
@@ -23,6 +23,9 @@ xpcc_gpio_enable(void)
 %%	endif
 %% endfor
 
+	// Enable Alternative Function clock
+	RCC->APB2ENR |= RCC_APB2ENR_AFIOEN;
+
```

Comment:
Can you take out the [corresponding line in `enableExternalInterrupt()`](https://github.com/strongly-typed/xpcc/blob/2aba3833e0221ce7ef077e7f0db7bf525d842358/src/xpcc/architecture/platform/driver/gpio/stm32f1/gpio.hpp.in#L234)?